src/index.coffee

```diff
@@ -109,6 +115,19 @@ class AlphabetizeKeys
     @_lintNodeKeys node, astApi, keys, 'Object'
 
 
+  _lintOverrideKeys: (node, astApi, keys) ->
+    {overrides} = astApi.config[@rule.name]
+    keys = keys.map @_stripQuotes
+    for key, index in keys
+      if index isnt 0 and overrides.indexOf(keys[index - 1]) > overrides.indexOf(key)
+        @errors.push astApi.createError {
+          lineNumber: node.locationData.first_line + 1
+          message: """
+            Keys should respect overrides ordering: #{keys[index - 1]} appears before #{key}
+          """
```

Comment:
Please tab in the `"""`
```coffee
message: """
  Keys should respect overrides ordering: #{keys[index - 1]} appears before #{key}
  """
```
The two are equivalent but I like that stylistically.